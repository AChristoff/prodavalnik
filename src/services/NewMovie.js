import React, { Component } from 'react'
import MovieForm from './MovieForm'

export default class NewMovie extends Component {
  render() {
    return (
      <div>
        <h2 data-testid="page-title">New Movie</h2>
        <MovieForm/>
      </div>
    )
  }
}
