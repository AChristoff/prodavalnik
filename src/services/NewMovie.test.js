import React from 'react';
import { render, cleanup, fireEvent } from '@testing-library/react';
import NewMovie from './NewMovie';

afterEach(cleanup);

test('<NewMovie>', () => {
  const { debug, getByTestId, queryByTestId } = render(<NewMovie/>);
  expect(getByTestId('page-title').textContent).toBe('New Movie');
  expect(queryByTestId('movie-form')).toBeTruthy();

  debug();
})