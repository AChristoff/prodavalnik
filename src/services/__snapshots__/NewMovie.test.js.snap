// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NewMovie> 1`] = `
<div>
  <h2
    data-testid="page-title"
  >
    New Movie
  </h2>
  <form
    data-testid="movie-form"
  >
    <label
      for="text"
    >
      Label
    </label>
    <input
      id="text"
      type="text"
    />
    <button
      type="submit"
    >
      Submit
    </button>
  </form>
</div>
`;
